push-subscription-management/index.js

```diff
@@ -1,34 +1,23 @@
-// [Working example](/push-subscription-management_demo.html).
```

Comment:
The changes in this recipe are not needed as part of this PR, right?
I'm not against them, but they should be made in a separate PR.